pcdsdevices/mirror.py

```diff
@@ -1478,3 +1478,5 @@ class for storing pitch positions based on state
 
     mr2l3_pitch_sic = Cpt(EpicsSignal, 'MR2L3:PITCH:Coating1')
     mr2l3_pitch_w = Cpt(EpicsSignal, 'MR2L3:PITCH:Coating2')
+    mr2l3_pitch_ccm_sic = Cpt(EpicsSignal, 'MR2L3:PITCH:CCM:Coating1')
+    mr2l3_pitch_ccm_w = Cpt(EpicsSignal, 'MR2L3:PITCH:CCM:Coating2')
```

Comment:
I think it'd be good if we started adding `doc=` indicators as to what these actually are. It can help in the auto-generated documentation as well as the typhos screens and such. That is to say:
```suggestion
    mr2l3_pitch_ccm_w = Cpt(EpicsSignal, 'MR2L3:PITCH:CCM:Coating2', doc="MR2L3 pitch coating 2 (Tungsten) setpoint position")
```

These may be immediately obvious to you, but perhaps not to the rest of the team.

Is that something you'd be willing to add in here?

Reply:
Seems reasonable to me, I'll update.